docs/en/sql-reference/syntax.md

```diff
@@ -58,10 +58,20 @@ Identifiers are:
 
 Identifiers can be quoted or non-quoted. The latter is preferred.
 
-Non-quoted identifiers must match the regex `^[a-zA-Z_][0-9a-zA-Z_]*$` and can not be equal to [keywords](#keywords). Examples: `x`, `_1`, `X_y__Z123_`.
+Non-quoted identifiers must match the regex `^[a-zA-Z_][0-9a-zA-Z_]*$` and cannot be equal to [keywords](#keywords).
+See the table below for examples of valid and invalid identifiers:
+
+| Valid identifiers                              | Invalid identifiers                    |
+|------------------------------------------------|----------------------------------------|
+| `xyz`, `_internal`, `Id_with_underscores_123_` | `1x`, `tom@gmail.com`, `äußerst_schön` |
```

Comment:
That looks äußerst schön, hehe.

Reply:
If we keep this up I'll be fluent in Deutsch from reading ClickHouse docs by the turn of the millennium! 😄